CHANGELOG.md

```diff
@@ -292,11 +292,13 @@
 - Handle attempts to activate invalid reports
 - `Recipient_region` codelist modified
 - `Intended beneficiaries` form field added, including validations, and country to region mapping
+<<<<<<< HEAD
```

Comment:
Looks like a merge conflict marker, just needs to be removed.

Reply:
not my day for this! Thanks!